src/Query/Grammar.php

```diff
@@ -10,6 +10,22 @@
 
 class Grammar extends MySqlGrammar
 {
+    /**
+     * Compile a "where fulltext" clause.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @param  array  $where
+     * @return string
+     */
+    public function whereFullText(Builder $query, $where)
+    {
+        $columns = $this->columnize($where['columns']);
+
+        $value = $this->parameter($where['value']);
+
+        return "MATCH ({$columns}) AGAINST (".$value.')';
```

Comment:
is there a difference between interpolating using `{$columns}` versus manual interpolation using `".$value."`?

Reply:
Bumping this question

Reply:
Not really, not in this use case. This is actually how [Laravel has it implemented for MySQL Grammar](https://github.com/laravel/framework/blob/master/src/Illuminate/Database/Query/Grammars/MySqlGrammar.php#L74), and I guess I initially took it from there.
But there's no real reason to use both interpolation methods here, will update to `{$value}` so it's visually cleaner